lib/zipline/zip_generator.rb

```diff
@@ -94,5 +95,10 @@ def is_active_storage_attachment?(file)
     def is_active_storage_one?(file)
       defined?(ActiveStorage::Attached::One) && file.is_a?(ActiveStorage::Attached::One)
     end
+
+    def is_url?(url)
```

Comment:
what is the `is_url?` method for?

Reply:
can you update the README.md with the changes you're recommending?

Reply:
So the idea here was to at least catch some common errors here instead of letting anything that is a string through.

But I'm open to suggestions!

As for the README, I'll try to update it soon, but I can't promise if it's gonna be this week.